Gemfile

```diff
@@ -7,7 +7,7 @@ gem 'gettext', '>= 3.1.3', '< 4.0.0'
 group :test do
   gem 'rake', '~> 10.1.0'
```

Comment:
I had a quick look at if this can be updated to a current version, but it blows up.